src/components/v5/common/ActionSidebar/partials/forms/SimplePaymentForm/hooks.ts

```diff
@@ -19,13 +21,34 @@ import { amountGreaterThanZeroValidation } from '~utils/validation/amountGreater
 import { hasEnoughFundsValidation } from '~utils/validation/hasEnoughFundsValidation.ts';
 import {
   ACTION_BASE_VALIDATION_SCHEMA,
+  CHAIN_FIELD_NAME,
   DECISION_METHOD_FIELD_NAME,
 } from '~v5/common/ActionSidebar/consts.ts';
 import useActionFormBaseHook from '~v5/common/ActionSidebar/hooks/useActionFormBaseHook.ts';
 import { type ActionFormBaseProps } from '~v5/common/ActionSidebar/types.ts';
+import {
+  type IconOption,
+  type SearchSelectOption,
+} from '~v5/shared/SearchSelect/types.ts';
 
 import { getSimplePaymentPayload } from './utils.tsx';
 
+export const useActiveChainsFilter = () => {
+  const activeProxyColoniesChainIds = useDeployedChainIds({
+    filterFn: (deployedProxyColony) => deployedProxyColony?.isActive,
+  });
+
+  const filterFn = ({ value: chainId }: SearchSelectOption<IconOption>) => {
+    // remove the default chain from the select
```

Comment:
I think here we should change the comment to `keep the default chain in the select`